sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -408,6 +409,21 @@ private static void resolveRangeOfRequiredSQVersion(Sonar sonar, List<Version> r
     return splitted;
   }
 
+  private static Version resolveLowVersion(String versionStr, String range) {
+    if (LATEST_KEYWORD.equals(versionStr)) {
+      throw new SonarVersionRangeException(String.format(
+        "Cannot use LATEST keyword at the start of a range in '%s'. Use 'sqVersions=LATEST' instead.",
+        range
+      ));
+    } else if (versionStr.endsWith("*")) {
```

Comment:
this check is particularly weak if parsing does not enforce `*` can only appear at the end of the version string

Reply:
Agreed. I'm not sure we should add a validation on the version format, to make sure we don't have funky versions like `7.*.1` (like you suggested in another comment).

I could move that validation to this class (I've kept the code on another branch). WDYT?

Reply:
given the complexity of `UpdateCenterDeserializer`, not matter how valuable I find this check to be, I personally wouldn't dare to change the class

Reply:
OK, leaving it like this then 🙂